pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/creator/SegmentGeneratorConfig.java

```diff
@@ -120,6 +120,18 @@ public enum TimeColumnType {
 
   private SegmentZKPropsConfig _segmentZKPropsConfig;
 
+  private final List<String> _bloomFilterCreationColumns = new ArrayList<>();
```

Comment:
(minor) Suggest moving these 2 list between `_invertedIndexCreationColumns` and `_textIndexCreationColumns`, also move the getters along with other getters

Reply:
Ack